#19238 |  Meta descriptions for dotnet wpf (#19238)
dotnet/docs: docs/framework/wpf/data/how-to-implement-property-change-notification.md

---
title: "How to: Implement Property Change Notification"

> description: Enable your binding target properties to automatically reflect the dynamic changes of the binding source in Windows Presentation Foundation (WPF).

Reviewer: I think this is a bit clearer.
```suggestion
description: Enable your properties to automatically notify a binding source when the property value changes in Windows Presentation Foundation (WPF).
```

Is it possible for you to also edit the intro paragraph? It seems completely incomprehensible. It looks like there was some update at some point that merged two intros.

I think that intro should change to:

```
To support <xref:System.Windows.Data.BindingMode.OneWay> or <xref:System.Windows.Data.BindingMode.TwoWay> binding and enable your binding target properties to automatically reflect the changes of the binding source .......
```
